dapps/assets/src/components/digitaltwins/AddDigitalTwin.ts

```diff
@@ -73,30 +79,31 @@ class AddDigitalTwinComponent extends mixins(EvanComponent) {
 
   async created() {
     this.runtime = await this.getRuntime();
-    dispatchers.twinCreateDispatcher.watch(({ detail: { status } }: CustomEvent) => {
-      if (status === 'starting') {
-        this.resetForm();
-        this.loading = false;
-
-        this.$toasted.show(
-          this.$t('_assets.digitaltwins.create-info'),
-          {
-            type: 'info',
-            duration: 10000, // 10 seconds
-          },
-        );
-      }
-
-      if (status === 'error') {
-        alert('There was a problem creating the twin. Please check your config and try again.');
-
-        this.loading = false;
-      }
-
-      if (status === 'finished' && typeof this.createdCallBack === 'function') {
-        this.createdCallBack();
-      }
-    });
+    this.createWatcher = dispatchers.twinCreateDispatcher
```

Comment:
Why isnt createWatcher a normal class method instead?

Reply:
The variable had the wrong naming. It represents the `clearWatcher` function, that is returned by the Dispatcher watch function.